grails-geb/README.md

```diff
@@ -161,6 +161,28 @@ To enable tracing, set the following system property:
   
 This allows you to opt in to tracing when an OpenTelemetry collector is available.
 
+#### GebConfig.groovy and using non-default browser settings
+Provide a `GebConfig.groovy` on the test runtime classpath (commonly `src/integration-test/resources`, but any location on the test classpath works) to customize the browser.
+
+To make this work, ensure:
+1. The `driver` property in your `GebConfig` is a `Closure` that returns a `RemoteWebDriver` instance.
+2. You set a custom `configuredBrowser` property so that `ContainerGebSpec` can start a matching container (e.g. "chrome", "firefox", "edge").
```

Comment:
as pointed out some projects potentially have GebConfig next to ContainerGebConfig. I saw other extensions always prefix their variables with their extension (e.g. sauceLabsBrowser), maybe we should too to avoid confusion? e.g. `containerBrowser` (in reference to containergebspec)